### DISCUSSION TITLE
vk still working even when blocked in blocked services

### DISCUSSION THREAD

**Discussion by ramialquraishi** [AdguardTeam/AdGuardHome#6089] [Q&A]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

I have installed Adguard on 2 rasp Pi on kali linux, I blocked VK in filters blocked settings and VK still works?

### Proposed solution

find one

### Alternatives considered and additional information

no workaround

**Comment by ainar-g** [comment]
Please do not use feature requests to report assumed bugs.  Provide the exact DNS query command that isn't working and make sure that you've actually set up your OS *and* your browser to query AdGuard Home.
